Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Header.css';
+
+function HeaderContent(props) {
+    return (
+      <React.Fragment>
+        <p className="headerText">{props.text}</p>
+      </React.Fragment>
```

Comment:
The same goes here as the last two comments. You could use object destructuring, but more importantly, the fragment is unnecessary. Just return the `p` tag.